scripts/ci/run_test.sh

```diff
@@ -1,40 +1,5 @@
 #!/usr/bin/env bash
 
 
-# This is a temporary fix to correct the KUBECONFIG file that gets mounted.
-# When the KUBECONFIG file is created by kind, it points to localhost. When inside
-# the cluster, we need to set this value to be .spec.clusterIP instead
-# See: https://github.com/operator-framework/operator-sdk/issues/2618
-KUBERNETES_SERVICE_HOST="$(kubectl get service kubernetes -o jsonpath='{.spec.clusterIP }')"
-temp=$(mktemp)
-cat "${KUBECONFIG}" | sed "s/server: https.*/server: https:\/\/${KUBERNETES_SERVICE_HOST}/g" > "${temp}"
-contents=$(<"${temp}")
-kubectl create cm kube-config --from-literal=kubeconfig="${contents}"
-rm "${temp}"
-
-# create roles and service account required for the test runner
-kubectl apply -f deploy/testrunner
-
-# start the test runner pod
-kubectl run test-runner --generator=run-pod/v1 \
-  --restart=Never \
-  --image-pull-policy=Always \
-  "--image=quay.io/mongodb/community-operator-testrunner:${version_id:?}" \
-  --serviceaccount=test-runner \
-  --command -- \
-  ./runner --operatorImage "quay.io/mongodb/community-operator-dev:${version_id}" \
-           --preHookImage "quay.io/mongodb/community-operator-pre-stop-hook:${version_id}" \
-           --testImage "quay.io/mongodb/community-operator-e2e:${version_id}" \
-           "--test=${test:?}"
-
-
-echo "Test pod is ready to begin"
-kubectl wait --for=condition=Ready pod -l run=test-runner --timeout=600s
-
-# The test will have fully finished when tailing logs finishes
-kubectl logs -f -l run=test-runner
-
-result="$(kubectl get pod -l run=test-runner -o jsonpath='{ .items[0].status.phase }')"
-if [[ ${result} != "Succeeded" ]]; then
-  exit 1
-fi
+pip3 install -r ./requirements.txt
+SKIP_CLEANUP="1" python3 ./scripts/dev/e2e.py --skip-operator-install 1 --skip-image-build 1 --test ${test:?} --tag ${version_id:?} --config_file ./scripts/ci/config.json --dump_diagnostic 1
```

Comment:
This SKIP_CLEANUP worries me a little bit. I don't see how this variable is used by the Python script but only by the Go tests. 

Can you explicitely pass this all the way to the test? This means basically to not use env variables